packages/lib/models/Setting.ts

```diff
@@ -1020,8 +1020,8 @@ class Setting extends BaseModel {
 				section: 'note',
 				public: false,
 				appTypes: [AppType.Mobile],
-				label: () => 'Opt-in to the editor beta',
-				description: () => 'This beta adds list continuation and syntax highlighting. If you find bugs, please report them in the Discourse forum.',
+				label: () => _('Opt-in to the editor beta'),
+				description: () => _('This beta adds list continuation and syntax highlighting. If you find bugs, please report them in the Discourse forum.'),
```

Comment:
Thank you for noticing this! I think some of these strings aren't user-facing, however.

`editor.beta` has `public: false`, so shouldn't be visible to the user. Thus, these strings shouldn't need to be translated.

Reply:
OK